arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Tuple5.kt

```diff
@@ -3,6 +3,9 @@
 
 package arrow.core
 
+import kotlin.jvm.JvmMultifileClass
+import kotlin.jvm.JvmName
```

Comment:
No, this is intentionally available in `kotlin-common` and it's meant so that you can influence the JVM output from common code. In this case make them appear as static functions on `Tuple5` instead of methods on the companion instance.

This is true for all annotations that can influence how binaries are generated from common code are available in `kotlin-common`, this is done to reduce the amount of code you need in platform specific functions. `JsName`, `JsModule`, `CName`, `Retain`, ...

Otherwise we'd need to make all `TupleN` classes `expect/actual` and basically duplicate the code accross all the platforms simply so we can annotate it with `@JvmName` to turn it into a static function instead of a method.

Reply:
@nomisRev I also think that was the case but I think @rachelcarmena found `@JvmInline` not working on the common folder for the refined types plugin

Reply:
Sorry, my fault! The reason comes from using `kotlin.stdlib.default.dependency=false`. I'll look for more info because it's being used in Arrow as well.

Thanks @nomisRev !! Resolved this conversation!!